### ISSUE TITLE
Dependencies not installed with the package via PyPi

### ISSUE THREAD

**Issue by themightywolfie** [HKUDS/LightRAG#333]
Installing the package via PyPi does not install the dependencies like aioboto3, torch, transformers, openai, ollama. nano_vectordb etc. I had to install each one individually. 

![image](https://github.com/user-attachments/assets/f90c4a17-c8ba-4741-ae4d-bab71ba4f305)

**Comment by aaqib-ahmed-nazir** [comment]
Yep had the same issue. Try downloading the dependency from the repo. 

#### Fix: 
- clone the repo first by:
```bash 
git clone https://github.com/HKUDS/LightRAG.git
```
- Then install it:
```bash 
cd LightRAG
pip install -e .
```

Will work :)
